addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,55 @@
+# iCloud Binding
+
+The iCloud Binding helps to integrate apple device information into openHAB.
+
+## Supported Things
+
+The following devices are known to work with this binding:
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* iPad Air 2/2017/Pro
+
+But other devices should work as well. Please provide feedback if you have tested another device type. We will add it to the list.
+
+## Discovery
+
+For each iCloud account to be included in openHAB a binding (bridge) needs to be configured with your apple id, password & refresh rate. 
+
+The devices registered to this account will then be automatically discovered.
+
+
+## Binding Configuration
+
+No textual configuration required/supported.
+
+## Thing Configuration
+
+Devices are automatically discovered once the binding (bridge) is configured.
```

Comment:
Textual configuration should still be supported and documented by every binding

Reply:
Which properties/parameters does the bridge/thing offer?

Reply:
I´ll have to look into how configuration files are supported; will take me a while. Then I´ll add the documentation.